Add página paciente.html

mariadxavier opened the pull request:
Página contém mostra de dados pessoais. Ainda a ser incluído: funcionalidades e aba de histórico.

mariadxavier reviewed (changes requested):
Acho que tivemos um ruído na comunicação. A Tag H1 deveria estar dentro da div criada anteriormente.

mariadxavier reviewed (approved):
Certo.

ojpbraga commented:
Gostaria de saber se foi suficiente a alteração realizada

ojpbraga commented:
[Prontuario.pdf](https://github.com/mariadxavier/prontuario-medico/files/14514611/Prontuario.pdf)

mariadxavier committed:
add paciente.html

mariadxavier committed:
add paciente.html

Página estilizada, ainda sem funcionalidades. Também há alterações no uso de header. Foi retirada a mesma do arquivo index.css.

mariadxavier committed:
fix style index.html

Alterações: uso do arquivo header.css criado por ojpbraga, remoção de itens desnecessários no menu nav, adição de imagem introdutória, alteração em medidas e configurações de estilo da mainDiv e adição de efeito  hover no botão de "ENTRAR".
